packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -275,16 +275,18 @@ export default defineComponent({
       focus()
     }
 
-    const showDropdown = () => {
+    const showDropdown = (forceFocus = false) => {
       if (props.disabled || props.readonly) { return }
       isOpenSync.value = true
       nextTick(() => {
-        timePicker.value?.focus()
+        if (!props.manualInput || forceFocus) {
+          timePicker.value?.focus()
+        }
       })
     }
 
-    const dropdownToggle = () => {
-      isOpenSync.value ? hideDropdown() : showDropdown()
+    const dropdownToggle = (forceFocus = false) => {
+      isOpenSync.value ? hideDropdown() : showDropdown(forceFocus)
```

Comment:
Just a thoughts - maybe in such cases it is better to just separate already existing logic instead of adding new one.

For example:
`showDropdown` currently focuses `dropdown`.

We can separate this to three methods.
`showDropdown`
`focusDropdown`
`showAndFocusDropdown`

And use this where it is needed.